src/context/schemas/context.schema.json

```diff
@@ -4,11 +4,13 @@
   "type": "object",
   "properties": {
     "type": { "type": "string" },
-    "name": { "type": "string" },
+    "name": { "type": "string" }, 
     "id": {
+      "$comment": "Reserved for identifiers",
       "type": "object",
-      "additionalProperties": { "type": "string" }
-    }
+      "additionalProperties": { "type": "any" }
```

Comment:
I don't think we can or should change the identifiers list be of type any, the idea has always been that they are strings.